Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -243,6 +243,14 @@ define([
          * @default 7500000.0
          */
         this.minimumTrackBallHeight = 7500000.0;
+        /**
+         * Keeps the camera above terrain when the camera is in a reference frame other than the world frame, i.e.
+         * the camera transform property is not the identity matrix. To re-orient the camera, it is assumed that the camera
+         * is viewing the center of the frame and the the up direction is aligned with the positive z axis in the local frame.
+         * @type {Boolean}
+         * @default true
+         */
+        this.avoidCollisionInReferenceFrame = true;
```

Comment:
I did this because people might not want us to reorient the camera after we detect a collision. For example, they may set the camera reference frame and not look at the center, then, when we detect a collision, it will look towards the center.

I though about adding another flag that enabled/disabled collision detection altogether, but would anyone really want to do that?